### PR TITLE
test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
- [DEMO LINK](https://OleksandrFEdev.github.io/layout_calendar/)
- [TEST REPORT LINK](https://OleksandrFEdev.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
1. New Days should be align by left side
image

2. if I change the modifier, then the day should appear in the correct column depending on the day
image

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Hi! I see the same problem. Plz make sure you have pushed all your commits
image

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
--

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
-

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
--

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
test

File: src/styles/blocks/caledar.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  justify-content: center;
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/caledar.scss:3]:
```suggestion
```
problem with layout not fixed.
to fix just remove justify-content
Screenshot from 2023-05-21 17-58-50

also `start-day` modifier does not work

File: src/styles/blocks/caledar.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    &:hover {
+      background-color: $color-pink;
+      transition: 0.5s;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+  }
+
+  &__day:first-child {
+    margin-left: ($size-day + $gap) * 6;
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/caledar.scss:28]: margin-left shouldn't be hardocoded and must depends on start-day modifier

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
the same issue
image

just ask for help in fe_chat

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
Previous review comments not fixed

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Hey, 
pls fix comments above!

If you have any questions - feel free to ask them in fe_chat

**Review by mykhalenych — APPROVED** [review]

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
test

**Review by mkyloa — APPROVED** [review]

**Review by volodymyr-soltys97 — APPROVED** [review]
test

**Review by slavanosatskyi — APPROVED** [review]

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!
We have some troubles with platform so we've got you HW several times, though you didn't make re-request. 
Please pay attention to previous mentor's comment about fixes implementation

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
Again

**Review by OleziO — APPROVED** [review]

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Still not fixed
2023-05-23_13h01_53

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Again test

**Pull request closed without merging**